blazor/dropdown-list/data-binding.md

```diff
@@ -9,66 +9,890 @@ documentation: ug
+
+In the following example, `Name` column from complex data has been mapped to the `Value` field.
+
+{% tabs %}
+{% highlight razor %}
+@using Syncfusion.Blazor.DropDowns
+
+<SfDropDownList TValue="string" TItem="Countries" Placeholder="e.g. Australia" DataSource="@Country">
+    <DropDownListFieldSettings Text="Name" Value="Code"></DropDownListFieldSettings>
 </SfDropDownList>
 
 @code {
 
-    public string DropVal;
-
     public class Countries
     {
         public string Name { get; set; }
 
         public string Code { get; set; }
     }
 
-    List<Countries> Country = new List<Countries>
-{
+    List<Countries>Country = new List<Countries>
+    {
         new Countries() { Name = "Australia", Code = "AU" },
         new Countries() { Name = "Bermuda", Code = "BM" },
         new Countries() { Name = "Canada", Code = "CA" },
         new Countries() { Name = "Cameroon", Code = "CM" },
+        new Countries() { Name = "Denmark", Code = "DK" },
+        new Countries() { Name = "France", Code = "FR" },
+        new Countries() { Name = "Finland", Code = "FI" },
+        new Countries() { Name = "Germany", Code = "DE" },
+        new Countries() { Name = "Greenland", Code = "GL" },
+        new Countries() { Name = "Hong Kong", Code = "HK" },
+        new Countries() { Name = "India", Code = "IN" },
+        new Countries() { Name = "Italy", Code = "IT" },
+        new Countries() { Name = "Japan", Code = "JP" },
+        new Countries() { Name = "Mexico", Code = "MX" },
+        new Countries() { Name = "Norway", Code = "NO" },
+        new Countries() { Name = "Poland", Code = "PL" },
+        new Countries() { Name = "Switzerland", Code = "CH" },
+        new Countries() { Name = "United Kingdom", Code = "GB" },
+        new Countries() { Name = "United States", Code = "US" },
     };
 }
-```
+{% endtabs %}
+{% endhighlight razor %}
+
+The output will be as follows.
+
+![Binding BLazor DropdownList Items](./images/blazor-dropdownlist-binding-items.png)
+
+## Primitive type
 
-## Index Value Binding
+You can bind the data to the DropDownList as a list of string, int, double and bool type items.
 
-Index value binding can be achieved by using `bind-Index` attribute and it supports int and int nullable types. By using this attribute you can bind the values respective to its index.
+The following code demonstrates array of string and integer values to the DropDownList component.
 
-```cshtml
+{% tabs %}
+{% highlight razor %}
 @using Syncfusion.Blazor.DropDowns
 
-<SfAutoComplete TValue="string" Placeholder="e.g. Australia" TItem="Countries" @bind-Index="@ddlIndex" DataSource="@Country">
-    <AutoCompleteFieldSettings Value="Name"></AutoCompleteFieldSettings>
-</SfAutoComplete>
+<SfDropDownList TValue="string" TItem="string" Placeholder="Select a game" DataSource="@data" @bind-Value="MyItem" Width="300px"></SfDropDownList>
+
+@code{
+    List<string> data = new List<string>() {"One", "Two", "Three"};
```

Comment:
There is no match between placeholder and data mapped to the component. Please ensure in all places
 
![image](https://user-images.githubusercontent.com/29191202/157011694-8eba920b-29bd-48d3-8f80-dad07d50a0ea.png)